front/package.json

```diff
@@ -85,6 +85,7 @@
     "not ie_mob 11",
     "not ios_saf 12.2-12.5",
     "not op_mini all",
+    "not kaios 2.5",
```

Comment:
Not sure it's related to this PR !

Reply:
So yes, it's part of this PR, but it's linked to your remark below on the useEffect.  Inside we have 'ResizeObserver' which is not supported by kios and caused an eslint error.  Checking what it was, I suggested not to put up with it.

Reply:
In the end I use the same method, so it still applies....